client/lib/directly/index.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+// import { loadScript } from 'lib/load-script';
+
+const DIRECTLY_SCRIPT_PATH = 'https://www.directly.com/widgets/rtm/embed.js';
+const DEFAULT_RTM_WIDGET_OPTIONS = {
+	// Automattic widget ID
+	id: '8a2968fc57d1e2f40157f42bf2d43160',
+	// By default don't show the "Ask a Question" button
+	displayAskQuestion: false
+};
+
+let initialized = false;
+
+const loadDirectlyScript = ( callback = noop ) => {
```

Comment:
@omarjackman I made the change to include `embed.js` in `lib/directly/vendor.js` as you suggested. The commit is here: https://github.com/Automattic/wp-calypso/pull/11306/commits/270bc34b3c43423713bc0e68cea058990a92ea37

I definitely think this leads to a better overall solution, so thanks for the suggestion. We can make small changes to the vendor JS that vastly simplify our implementation (no more `id="directlyRTMScript"` issue), and it keeps all the vendor quirks away from the API into the service.

Now I gotta go fix all the tests I just broke. :)